resemblyzer/voice_encoder.py

```diff
@@ -9,38 +9,43 @@
 
 
 class VoiceEncoder(nn.Module):
-    def __init__(self, device: Union[str, torch.device]=None, verbose=True):
+    def __init__(self, device: Union[str, torch.device]=None, verbose=True, weights_fpath: Union[Path, str]=None):
         """
-        :param device: either a torch device or the name of a torch device (e.g. "cpu", "cuda"). 
+        :param device: either a torch device or the name of a torch device (e.g. "cpu", "cuda").
+        :param weights_fpath: path to "<CUSTOM_MODEL>.pt" file path
```

Comment:
precise that leaving it as none will default to the default model

Reply:
Sure

Reply:
I don't see your changes, did you push?